sdcm/sct_config.py

```diff
@@ -2413,8 +2415,23 @@ def get_version_based_on_conf(self):  # pylint: disable=too-many-locals
             _is_enterprise = 'enterprise' in docker_repo
         self.scylla_version = scylla_version
         self.is_enterprise = _is_enterprise
+        self.update_argus_with_version(scylla_version, "scylla-server")
```

Comment:
@k0machi we need to align what exactly are the name we want to pass for each...

Reply:
This will submit the target package, if you want this version visible to be seen on the test dashboard you need to use the `ArgusSCTClient.update_scylla_version` method for that.